notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<p>Consider removing the output after review is finished. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Cleared in the new commit :)

Reply:
<p>1,16 is relatively small input sequence, can we use another value e.g. 1,128 or provide some hint to users how to change sequence size?</p>

Reply:
The benefits from sparsity can be different at different sequence lengths. We can try 128, for sure if it leads to similar performance improvements. If not I would suggest keeping something moderate, e.g. 32 or 64.

Reply:
I think at least provide some description that this parameter may be vary and what need to be done to modify benchmarking line to select different len will be good mention.

Reply:
Changed to 64. Also mentioned that users are recommended to tune based on their applications. See new commit. Thanks.